x-pack/plugins/observability/public/components/shared/page_template/nav_name_with_technical_preview.tsx

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
I think it will be nice to have one component showing the beta badge and reuse it if possible. The component can receive different props - for example, the difference with the technical preview is the icon which can be an optional prop and the label can be passed as prop as well. What do you think?